rfcs/convergence/exposing-popper-options.md

```diff
@@ -0,0 +1,224 @@
+  {...props} // ICalloutProps interface
+/>
+
+<ContextualMenu
+  calloutProps={calloutProps} // ICalloutProps interface
+  // also can spread calloutProps
+  calloutMaxHeight
+  bounds
+  directionalHintFixed
+/>
+```
+
+v0:
+
+```tsx
+<Popup
+  align
+  position
+  offset
+/>
+
+<Menu
+  items={[
+    { menu: popper: { align, position, offset }}
+  ]}
+/>
+```
+
+## Problem statement
+
+There is a subset of `usePopper` options that are currently used by all converged components that need positioning.
+These options are spread into the component's props:
+
+```tsx
+<Tooltip position="bottom" align="center" />
+<Popover position="bottom" align="center" />
+<Menu position="bottom" align="center" />
+```
+
+There are other popper options that are used in some components but not others, examples of these options are:
+
+- `coverTarget` - allows the positioned element to cover the target
+
+There are also additional options that are not used by any converged component that might be used in the future like:
+
+- `autoSize` - Allows the positioned element to change size based on viewport size
+- `flipBoundary` - Determines what bounds to use to flip the positioning
+
+Issues and bugs can be triggered because certain use cases require more precise configuration of popper. The most likely
+result would be that a popper configuration is exposed for one component but not others.
+
+## Detailed Design or Proposal
+
+### Equal popper configuration
+
+Each converged component that uses popper should allow an equal level of configuration. This ensures that our positioned
+components can behave consistently across the library. Any fixes that require exposing another popper configuration will
+be available to **all** components that use popper.
+
+This proposal does not mean that all popper options must be public, but that if one component exposes a popper configuration,
+then all components should do so equally.
+
+We can achieve this with an agreement that the current `PositioningProps` would be supported by all v9 components, while
+internal popper options can still belong to `PopperOptions` type in the `react-positioning` package.
+
+### Shorthand `placement` configuration
+
+The next step to this proposal is to make sure that all components which use a positioned element should allow users
+to configure popper through a `placement` prop that behaves like a shorthand. An extra helper will be required to resolve
+shorthand to full popper configuration internally, but the external API for `placement` is very clean for users.
+
+The usage examples below will detail the usage of this shorthand.
+
+```tsx
+// @fluentui/react-positioning
+export type PositioningShorthandValue =
+  | 'above'
+  | 'above-start'
+  | 'above-end'
+  | 'below'
+  | 'below-start'
+  | 'below-end'
+  | 'before'
+  | 'before-top'
+  | 'before-bottom'
+  | 'after'
+  | 'after-top'
+  | 'after-bottom';
+
+export type PositioningShorthand = PositioningProps | PositioningShorthandValue;
```

Comment:
Should we consider renaming these types to align with the `placement` prop name?
`PlacementProps`
`PalcementShorthandValue`
`PlacementShorthand`

Reply:
well my original suggestion was `positioning` :)